### ISSUE TITLE
Testing structure for app

### ISSUE THREAD

**Issue by nscharioth** [verdigado/gruene-app#326]
As an app developer I would like to have a ready and efficient testing structure, which allows the easy testing of new application versions.

Acceptance criteria:
- [ ] testing methodology ready and usable

Pollion reference:
https://github.com/pollion/sunflower-deployment/issues/23

**Comment by nscharioth** [comment]
Review issue description and move to "Ready for Development". @leagerndt @JuliaFreitagBGSt

**Comment by nscharioth** [comment]
I would suggest we move the testing structure to non-critical. Please review @leagerndt @JuliaFreitagBGSt

**Comment by JuliaFreitagBGSt** [comment]
@nscharioth agreed. As discussed we will stick to human testing for now and develop a testing structure in the future
@leagerndt
